resdk/resolwe.py

```diff
@@ -44,6 +44,18 @@
 TOOLS_REMOTE_HOST = os.environ.get('TOOLS_REMOTE_HOST', None)
 
 
+class ResolweResource(slumber.Resource):
```

Comment:
I worry this may be mistaken for resource classes in the resources module. It is probably OK, it fits nicely with the other ResolweXXX classes.